### PR TITLE
add wasmer 1.0.1 for Linux and Windows

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [ ] Package does not vendor other packages. 
  - [x] (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
  - [x] some ATTRIBUTIONS are included
  - [x] all licenses are checked with `cargo-license`
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes
- reboot of #9311
- [x] licenses
  - [ ] would be helped by https://github.com/conda-forge/staged-recipes/pull/11844
- [x] linux
- [ ] ~~macos~~
  - [ ] ~~might have upstream issues~~
  - _I don't know how to fix this, PRs welcome! :shrug: _
- [x] windows
  - [ ] moving to `%SCRIPTS%`.... would `%PREFIX%/Library/bin/` be more appropriate?
- getting _the baseline runtime_ up is better than nothing

## After this is merged
- There are a number of additional outputs this tarball _could_ create 
  - `wapm`
  - the C API
  - the new "lightweight" runtime
  - lighter-weight builds with fewer compiler/feature bells-and-whistles
- some of the above would be needed for e.g. the python integration, which is what @conda-forge/fastdiff needs

### CODE REVIEW

**Comment by bollwyvl** [comment]
Hey, one :heavy_check_mark: is better an :zero: !

**Comment by bollwyvl** [comment]
current OSX adventure, needs research:
```
   Compiling blake3 v0.3.7
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-m64" "-arch" "x86_64" "-L" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.0.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.1.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.2.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.3.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.4.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.5.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.6.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.7.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.build_script_build.2s310o1t-cgu.8.rcgu.o" "-o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/wasmer-vm-2ada8141d1afa805/build_script_build-2ada8141d1afa805.39ekdsnb12i69fw5.rcgu.o" "-Wl,-dead_strip" "-nodefaultlibs" "-L" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/deps" "-L" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/deps/libcc-0f4c7c727f25b73e.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libstd-518979da66993550.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-f52081aedccc205f.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libobject-bad1c02788187328.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libaddr2line-6375d8dbf53aee9d.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libgimli-0ea8539778b0c0ab.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/librustc_demangle-64f30c6d69babb18.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libhashbrown-e27428d997fd2532.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_alloc-f02467a76e5a8054.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libunwind-9536efad7116d828.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libcfg_if-5ca171f6ec11d7a5.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/liblibc-14b621861b38e51f.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/liballoc-38f4a0c206512fa5.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_core-99bca6ace58141c5.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libcore-72a66f4c97a4c0c8.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-e7cd9f0beed2fc0f.rlib" "-lSystem" "-lresolv" "-lc" "-lm" "-Wl,-rpath,/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib"
  = note: Undefined symbols for architecture x86_64:
            "_linkat", referenced from:
                std::sys::unix::fs::link::had1f447ad1437b53 in libstd-518979da66993550.rlib(std-518979da66993550.std.byn0lf9c-cgu.0.rcgu.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: aborting due to previous error

error: could not compile `wasmer-vm`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-m64" "-arch" "x86_64" "-L" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.0.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.1.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.10.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.11.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.12.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.13.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.14.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.15.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.2.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.3.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.4.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.5.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.6.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.7.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.8.rcgu.o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.build_script_build.5huingkf-cgu.9.rcgu.o" "-o" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/build/blake3-fb6ae7a51cf38d57/build_script_build-fb6ae7a51cf38d57.46vi5hgwmqb81aj.rcgu.o" "-Wl,-dead_strip" "-nodefaultlibs" "-L" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/deps" "-L" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target/release/deps/libcc-0f4c7c727f25b73e.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libstd-518979da66993550.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-f52081aedccc205f.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libobject-bad1c02788187328.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libaddr2line-6375d8dbf53aee9d.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libgimli-0ea8539778b0c0ab.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/librustc_demangle-64f30c6d69babb18.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libhashbrown-e27428d997fd2532.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_alloc-f02467a76e5a8054.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libunwind-9536efad7116d828.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libcfg_if-5ca171f6ec11d7a5.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/liblibc-14b621861b38e51f.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/liballoc-38f4a0c206512fa5.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_core-99bca6ace58141c5.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libcore-72a66f4c97a4c0c8.rlib" "/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_build_env/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-e7cd9f0beed2fc0f.rlib" "-lSystem" "-lresolv" "-lc" "-lm" "-Wl,-rpath,/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib"
  = note: Undefined symbols for architecture x86_64:
            "_linkat", referenced from:
                std::sys::unix::fs::link::had1f447ad1437b53 in libstd-518979da66993550.rlib(std-518979da66993550.std.byn0lf9c-cgu.0.rcgu.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: aborting due to previous error

error: failed to compile `wasmer-cli v1.0.0 (/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/lib/cli)`, intermediate artifacts can be found at `/Users/runner/miniforge3/conda-bld/wasmer_1609969438370/work/target`


```

**Comment by bollwyvl** [comment]
Kicking again, as it might have been all upstream...
- https://github.com/conda-forge/rust-feedstock/issues/73
  - https://github.com/conda-forge/admin-requests/pull/212

**Comment by bollwyvl** [comment]
new osx fail:
```
= note: Undefined symbols for architecture x86_64:
            "_clock_gettime", referenced from:
                wasmer_wasi::syscalls::clock_time_get::h9ead72cf0f39937a in libwasmer_wasi-7317f238f6c0a952.rlib(wasmer_wasi-7317f238f6c0a952.wasmer_wasi.dth3jwkl-cgu.0.rcgu.o)
               (maybe you meant: wasmer_emscripten::time::_clock_gettime::h76c59fc810177ad6)
            "_clock_getres", referenced from:
                wasmer_wasi::syscalls::clock_res_get::h38975da4bc128a14 in libwasmer_wasi-7317f238f6c0a952.rlib(wasmer_wasi-7317f238f6c0a952.wasmer_wasi.dth3jwkl-cgu.0.rcgu.o)
               (maybe you meant: wasmer_emscripten::time::_clock_getres::heca3955820e30034)
            "_fdopendir$INODE64", referenced from:
                wasmer_emscripten::syscalls::unix::___syscall220::hf2264ad8ce9a5cb6 in libwasmer_emscripten-4178b38084047f6f.rlib(wasmer_emscripten-4178b38084047f6f.wasmer_emscripten.e40rwe6n-cgu.5.rcgu.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

will just try an even older rust there...

**Comment by isuruf** [comment]
That's not a rust issue. It's `wasmer` which needs a newer macos.

**Comment by bollwyvl** [comment]
Very good! I'm out of my depth on several dimensions on this one. If it's going to be complicated like this frequently, I'm probably not the right maintainer for the job!

**Comment by wolfv** [comment]
should we merge wasmer with only linux buidls for now, and experiment with Windows and OSX support in the feedstock? I am happy to help here @bollywyl (I already put emscripten on conda-forge, in order to compile wasm binaries).

**Comment by wolfv** [comment]
(we can request a new osx sdk with some variables https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks but I am not sure this will solve this problem here, since Rust might be different).

**Comment by bollwyvl** [comment]
>  with only linux buidls

I think it's probably worth a _shot_ on windows... two platforms is still cross-platform! I just kind of lost heart :broken_heart: I've generally been impressed with how straightforward the rust builds are on `conda-forge`, but i'm always chasing the new "best practices": currently, I think [py-spy](https://github.com/conda-forge/py-spy-feedstock) probably has the best stuff to steal.

> emscripten on conda-forge, in order to compile wasm binaries

hey, awesome!

I think a big part of the automation of delivering wasm at scale will be the testability, e.g. how hard will it be to make an automated feedstock PR that:
- compiles a package with `emscripten` :tada:
- tests it in `firefox` under `selenium` :tada:
- tests it in `wasmer` :pensive: 

If it has to be a whole other feedstock that can't be _entirely_ automated from the upstream, I don't know if this is going to fly...

**Comment by wolfv** [comment]
sure windows would be great! 

I have the `boa-forge` project to experiment with new-style recipes and additionally try to build a more minimal micromamba. I was thinking of experimenting with emscripten support over there and see how far we could get towards a micromamba that runs in wasmer :)

https://github.com/mamba-org/boa-forge

**Comment by bollwyvl** [comment]
Aw, man, please PLEASE call it `microwamba`... it's like a smaller, more ill-mannered

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: recipes/wasmer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    Wasmer is a standalone JIT WebAssembly runtime, aiming to be fully
+    compatible with WASI and Emscripten.
+  dev_url: https://github.com/wasmerio/wasmer
+  doc_url: https://docs.wasmer.io
+  doc_source_url: https://github.com/wasmerio/docs.wasmer.io
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - wolfv
```
Comment (bollwyvl) [inline recipes/wasmer/meta.yaml:89]: @wolfv can I get a confirmation here?

Also, if you are (or anybody is!) dissatisfied with my pace of development or particular choices, please feel free to either:
- brutally review
- PR (or push at, i guess if you're CF core :laughing:) at this
- Start a new PR, and we can trade back and forth

Comment (wolfv) [reply recipes/wasmer/meta.yaml:89]: sure, confirming! I am really happy with everything you do ;) keep on pushing!

File: recipes/wasmer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  # TODO: determine if we can use a newer OSX, or somehow push upstream on the missing pieces
+  skip: true  # [osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - rust >=1.47  # [not osx]
+    - rust ==1.47  # [osx]
```
Comment (isuruf) [inline recipes/wasmer/meta.yaml:22]: Use `compiler('rust')`

**Comment by wolfv** [comment]
noice! :tada:

**Comment by bollwyvl** [comment]
Well, that's... Something! There are a lot of additional packaging choices
from this first tarball, like wap,  the C API, and the "mini" runtime, but
I think what we have in the can "runs wasm," which is what we probably need
first to start exploring.

I need to do another run through for self review, and of course probably
forgot...

@conda-forge-admin please rerender

But otherwise interested in any feedback from @conda-forge/staged-recipes
or @conda-forge/help-rust (if that is a thing)

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes ready for review

cc @wolfv

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by wolfv** [comment]
btw how do you generate the licenses? I sure hope you didnt add them manually to the recipe :)

**Comment by bollwyvl** [comment]
I _find I'm missing them_ automatically, then have some scripts that use
the JSON to candidates for missing ones and review them. But yeah, it's
pretty BAD. I'm sort of surprised, given how nice cargo seems otherwise.

**Pull request merged**